dev/content/Composables.vue

```diff
@@ -0,0 +1,187 @@
+<script setup lang="ts">
+import { RequestOptions } from "@/api/base"
+import { AxiosError } from "axios";
+import { computed } from "vue"
+import useBaseAPI from "../../src/composables/useBaseAPI"
+import { debounceLeading } from "../../src/helpers/Debounce"
+
+interface Thing {
```

Comment:
Motion to change this to some coniferous tree related analogy, like `Fir` or `Spruce`.